src/derive/curve.rs

```diff
@@ -99,26 +117,68 @@ macro_rules! new_curve_impl {
                     }
                 }
 
+                impl GroupEncoding for $name {
+                    type Repr = [< $name Compressed >];
+
+                    fn from_bytes(bytes: &Self::Repr) -> CtOption<Self> {
+                        $name_affine::from_bytes(bytes).map(Self::from)
+                    }
+
+                    fn from_bytes_unchecked(bytes: &Self::Repr) -> CtOption<Self> {
+                        $name_affine::from_bytes(bytes).map(Self::from)
+                    }
+
+                    fn to_bytes(&self) -> Self::Repr {
+                        $name_affine::from(self).to_bytes()
+                    }
+                }
+
+
+                // The flags are placed in the last byte, the MSB.
+                #[allow(non_upper_case_globals)]
+                const [< $name _FLAG_BYTE_INDEX>]: usize= [< $name _COMPRESSED_SIZE >]-1 ;
+
+                #[allow(non_upper_case_globals)]
+                const [< $name _FLAG_BITS >]: u8 =
```

Comment:
This is called `name_FLAG_BITS` but contains `name_not_FLAG_BITS`.
I would suggest two options:
- a. Change the name
- b. Keep the name but flip the bits, and then to Clear flag bits:
`tmp[name_FLAG_BYTE_INDEX] &= !name_FLAG_BITS`